pacai/ui/topGraphics.py

```diff
@@ -0,0 +1,600 @@
+                            add(screen, (self.gridSize * (-0.5), -1 * wallSize)), wallColor)
+                    graphicsUtils.line(add(screen, (-1 * wallSize, -2 * wallSize + 1)),
+                            add(screen, (-1 * wallSize, self.gridSize * (-0.5))), wallColor)
+
+                # SE quadrant
+                if (not sIsWall) and (not eIsWall):
+                    # inner circle
+                    graphicsUtils.circle(screen2, wallSize, wallColor, wallColor, (270, 361), 'arc')
+                if (sIsWall) and (not eIsWall):
+                    # vertical line
+                    graphicsUtils.line(add(screen, (wallSize, 0)),
+                            add(screen, (wallSize, self.gridSize * (0.5) + 1)), wallColor)
+                if (not sIsWall) and (eIsWall):
+                    # horizontal line
+                    graphicsUtils.line(add(screen, (0, wallSize)),
+                            add(screen, (self.gridSize * 0.5 + 1, wallSize)), wallColor)
+                if (sIsWall) and (eIsWall) and (not seIsWall):
+                    # outer circle
+                    graphicsUtils.circle(add(screen2, (2 * wallSize, 2 * wallSize)),
+                            wallSize - 1, wallColor, wallColor, (90, 181), 'arc')
+                    graphicsUtils.line(add(screen, (2 * wallSize - 1, wallSize)),
+                            add(screen, (self.gridSize * 0.5, wallSize)), wallColor)
+                    graphicsUtils.line(add(screen, (wallSize, 2 * wallSize - 1)),
+                            add(screen, (wallSize, self.gridSize * (0.5))), wallColor)
+
+                # SW quadrant
+                if (not sIsWall) and (not wIsWall):
+                    # inner circle
+                    graphicsUtils.circle(screen2, wallSize, wallColor, wallColor, (180, 271), 'arc')
+                if (sIsWall) and (not wIsWall):
+                    # vertical line
+                    graphicsUtils.line(add(screen, (-1 * wallSize, 0)),
+                            add(screen, (-1 * wallSize, self.gridSize * (0.5) + 1)), wallColor)
+                if (not sIsWall) and (wIsWall):
+                    # horizontal line
+                    graphicsUtils.line(add(screen, (0, wallSize)),
+                            add(screen, (self.gridSize * (-0.5) - 1, wallSize)), wallColor)
+                if (sIsWall) and (wIsWall) and (not swIsWall):
+                    # outer circle
+                    graphicsUtils.circle(add(screen2, (-2 * wallSize, 2 * wallSize)),
+                            wallSize - 1, wallColor, wallColor, (0, 91), 'arc')
+                    graphicsUtils.line(add(screen, (-2 * wallSize + 1, wallSize)),
+                            add(screen, (self.gridSize * (-0.5), wallSize)), wallColor)
+                    graphicsUtils.line(add(screen, (-1 * wallSize, 2 * wallSize - 1)),
+                            add(screen, (-1 * wallSize, self.gridSize * (0.5))), wallColor)
```

Comment:
Yes, so here the line break is different. So one of the two needs to be changed.